IBPSA/Examples/Tutorial/SimpleHouse/SimpleHouseTemplate.mo

```diff
@@ -0,0 +1,103 @@
+  Modelica.Thermal.HeatTransfer.Components.ThermalResistor wallRes(
+    R=d_wall/A_wall/k_wall)
+    "Thermal resistor for wall: 25 cm of rockwool"
+    annotation (Placement(transformation(extent={{80,-10},{100,10}})));
+equation
+  connect(weaDat.weaBus, weaBus) annotation (Line(
+      points={{-180,-10},{-150,-10}},
+      color={255,204,51},
+      thickness=0.5));
+  connect(Tout.T, weaBus.TDryBul)
+    annotation (Line(points={{-22,0},{-150,0},{-150,-10}},color={0,0,127}));
+  connect(Tout.port,wallRes. port_a)
+    annotation (Line(points={{0,0},{80,0}},          color={191,0,0}));
+  annotation (Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-240,
+            -220},{200,200}}), graphics={
+        Rectangle(
+          extent={{-220,40},{20,-40}},
+          fillColor={238,238,238},
+          fillPattern=FillPattern.Solid,
+          pattern=LinePattern.None),
+        Rectangle(
+          extent={{-220,-60},{180,-200}},
+          fillColor={238,238,238},
+          fillPattern=FillPattern.Solid,
+          pattern=LinePattern.None),
+        Rectangle(
+          extent={{-220,180},{180,60}},
+          fillColor={238,238,238},
+          fillPattern=FillPattern.Solid,
+          pattern=LinePattern.None),
+        Rectangle(
+          extent={{40,40},{180,-40}},
+          fillColor={238,238,238},
+          fillPattern=FillPattern.Solid,
+          pattern=LinePattern.None),
+        Text(
+          extent={{98,20},{32,38}},
+          textColor={0,0,127},
+          fillColor={255,213,170},
+          fillPattern=FillPattern.Solid,
+          textString="Wall"),
+        Text(
+          extent={{-148,-86},{-214,-68}},
+          textColor={0,0,127},
+          fillColor={255,213,170},
+          fillPattern=FillPattern.Solid,
+          textString="Heating"),
+        Text(
+          extent={{-118,18},{-214,40}},
+          textColor={0,0,127},
+          fillColor={255,213,170},
+          fillPattern=FillPattern.Solid,
+          textString="Weather inputs"),
+        Text(
+          extent={{-76,158},{-214,180}},
+          textColor={0,0,127},
+          fillColor={255,213,170},
+          fillPattern=FillPattern.Solid,
+          textString="Cooling and ventilation")}),
+    experiment(Tolerance=1E-6, StopTime=1e+06),
+    __Dymola_Commands(file=
+          "modelica://IBPSA/Resources/Scripts/Dymola/Examples/Tutorial/SimpleHouse/SimpleHouseTemplate.mos"
+        "Simulate and plot"),
+    Documentation(revisions="<html>
+<ul>
+<li>
+September 4, 2023, by Jelger Jansen:<br/>
+Replace IDEAS by IBPSA models.
+</li>
+<li>
+October 11, 2016, by Filip Jorissen:<br/>
+First implementation.
+</li>
+</ul>
+</html>", info="<html>
+<p>
+This model is used as the starting point for the <code>SimpleHouse</code> tutorial.
+It was copied from the Modelica crash course organised by KU Leuven
+(<a href=\"https://github.com/open-ideas/__CrashCourse__\">https://github.com/open-ideas/__CrashCourse__</a>).
```

Comment:
This documentation should describe what this model implements. If the `resWall` stays here, the info-section from `SimpleHouse1` should be added

Reply:
I added a sentence explaining the content of this start file. Since they don't have to model this themselves, I didn't add a section on 'required models' and/or 'connection instructions'.